git_machete/utils.py

```diff
@@ -94,7 +94,21 @@ def debug(msg: Optional[str] = None) -> None:
     if debug_mode:
         function_name = bold(inspect.stack()[1].function)
         args, _, _, values = inspect.getargvalues(inspect.stack()[1].frame)
-        function_args = bold(f"({', '.join([arg+'='+str(values[arg]) if arg != 'self' else 'self' for arg in args])})")
+
+        redact_arg_by_name = ['token']
+        for arg in redact_arg_by_name:
+            values[arg] = '***'
+
+        redact_arg_by_value = ['ghp_']
+        for arg, value in values.items():
+            if value and isinstance(value, str):
+                if any(value.startswith(arg_) for arg_ in redact_arg_by_value):
```

Comment:
I'd even make it more strict, sth like
```suggestion
              if any(arg_ in str(value) for arg_ in redact_arg_by_value):
```